modules/ROOT/pages/split-aggregate-scope.adoc

```diff
@@ -0,0 +1,72 @@
+= Split Aggregate Scope
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: anypoint studio, studio, mule, split, aggregate, scope
+
+The Split Aggregate scope splits a collection and processes each part with the components within it in parallel (subject to any limitation in concurrency that has been configured).
+After all messages have been processed they will be aggregated before continuing with the flow in the same order they were before splitting the defined collection. 
+
+The Split Aggregate scope provides fields described in this table:
+
+[%header,cols="1,3"]
+|===
+|Attribute |Description
+|Collection (`collection`) | Specifies the expression that define the collection of parts to be processed in parallel. By default, it will use the incoming payload.
+|Timeout (`timeout`) | Specifies the timeout for each parallel route. The default behaviour is that of no timeout set.
+|Max Concurrency (`maxConcurrency`) | Specifies the maximum level of parallelism that will be used by this router. By default, all routes will run in parallel.
+|Target Variable (`target`) | Specifies the variable where to save the processed payload. 
+|Target Value (`targetValue`) | Specifies an expression to evaluate against the operation's output value. The outcome of this expression will be stored in the target variable. 
+|===
+
+This XML example adds to every element in the collection the string `"-result"`: 
+
+----
+<split-aggregate>
+	<collection>
+		['apple', 'banana', 'orange']
+	</collection>
+	<set-payload value="#[payload ++ '-result']"/>
+</split-aggregate>
+<!-- payload will now be ['apple-result', 'banana-result', 'orange-result']
+----
+
+The example above is equivalent to: 
+
+----
+<split-aggregate collection="#[['apple', 'banana', 'orange']]">
+	<set-payload value="#[payload ++ '-result']"/>
+</split-aggregate>
+----
+
+The next example shows that the variables can also be aggregated:
```

Comment:
Give to this some header and more explanation.
what happens if already exist a variable before with the same name?